src/main/resources/application.yaml

```diff
@@ -218,3 +218,7 @@ process-payments:
   enabled: ${PROCESS_PAYMENTS_ENABLED:true}
 
 use-wrapping-zip: true
+
+service-config:
```

Comment:
is it a service-specific config?

Reply:
no, it is not. is there a better config for this situation?

Reply:
you can make it a flat property.
maybe `ocr-valiration-max-tries`